pom.xml

```diff
@@ -79,12 +79,11 @@
   </distributionManagement>
 
   <properties>
-    <mavenVersion>3.6.3</mavenVersion>
+    <mavenVersion>3.9.7</mavenVersion>
```

Comment:
this doesn't look related to the subject of the PR
maybe change the subject

Reply:
changed the subject